flutter_local_notifications/lib/src/platform_specifics/android/notification_channel_output.dart

```diff
@@ -0,0 +1,17 @@
+import 'enums.dart';
+
+/// Settings for Android notification channels.
+class AndroidNotificationChannelOutputInfo {
```

Comment:
There's already an `AndroidNotificationChannel` class for notification channels, which should be used instead to avoid confusion. The only reason I can see why another class was not this PR hasn't retrieved all of the information for the other properties. From what I can see [here](https://developer.android.com/reference/android/app/NotificationChannel), it should be possible to get the remaining info